server/dataone_register.py

```diff
@@ -144,13 +144,12 @@ def find_initial_pid(path):
     if re.search(r'^http[s]?:\/\/search.dataone.org\/#view\/', path):
         return re.sub(
             r'^http[s]?:\/\/search.dataone.org\/#view\/', '', path)
-    elif re.search(r'^http[s]?://cn.dataone.org/cn/d1/v[\d]/\w+/', path):
+    elif re.search(r'^http[s]?:\/\/cn.dataone.org\/cn(\/d1)?\/v[\d]/\w+\/', path):
```

Comment:
same comment as my earlier one about the `d1` in the middle of the URL. Do you have an example of this? Maybe I'm just having a brain fart.